doc/cprover-manual/contracts-quantifiers.md

```diff
@@ -0,0 +1,43 @@
+# Quantifiers
+
+### Syntax
+
+```c
+__CPROVER_forall { *identifier* *type*; *boolean expression* }
+__CPROVER_exists { *identifier* *type*; *boolean expression* }
+```
+
+Both quantifiers are supported in function-contracts context.
+
+
```

Comment:
May be here we can say:

---

While quantified expressions with arbitrary Boolean expressions are allowed with an SMT backend, the SAT backend only supports bounded quantification under _constant_ lower & upper bounds. This is because the SAT backend currently expands a universal quantifier (`__CPROVER_forall`) to a conjunction and an existential quantifier (`__CPROVER_exists`) to a disjunction on each value within the specified bound.

Concretely, with the SAT backend, the following syntax must be used for quantifiers within function contracts:

```
__CPROVER_forall { *id* *type*; *range* => *boolean expression* }
__CPROVER_exists { *id* *type*; *range* && *boolean expression* }
```

where `*range*` is an expression of the form

```
*lower bound* <= *id* && *id* <= *upper bound*
```

where `*lower bound* and `*upper bound*` being constants.

Reply:
Implemented as suggested. Thanks!